docs/dev_sliding_dot_product.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>2^15 seems short. Though, I don't know what is a "good" test. Maybe we should look for other FFT benchmarks to compare against</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>There was no particular rationale behind <code>2^15</code>. I just wanted a time series that can reflect the differences from one version to another to some extent. If I choose a short time series, I cannot see those differences easily.</p><p> </p><p>After selecting a version, we can then compare it with the reference for time series with different lengths, e.g <code>np.power(2, np.arange(10, 21)).</code></p><p>This comparison is provided at the bottom of the notebook.</p><p> </p><p>We may also get some idea from the OTFFT benchmark provided here: <a href="http://wwwa.pikara.ne.jp/okojisan/otfft-en/index.html#singlethread" target="_blank">http://wwwa.pikara.ne.jp/okojisan/otfft-en/index.html#singlethread</a>  </p>